### PR TITLE
Update 1.14 behavioral changes to mention...

### PR DESCRIPTION
**Pull request by bananabowl**
tf.keras.optimizers.Adadelta's default learning rate change

### CODE REVIEW

**Review by tanzhenyu — COMMENTED** [review]
Thanks for the PR, meanwhile would you mind adding the following as well:
1. Adagrad changed from 0.01 to 0.001
2. Adamax changed from 0.002 to 0.001
3. NAdam changed from 0.002 to 0.001
4. Adadelta changed from 1.000 to 0.001

Thanks!

**Review by gunan — DISMISSED** [review]

**Comment by gunan** [comment]
Why do we have the change against master?
I thought these were for the patch release?

**Comment by bananabowl** [comment]
> Thanks for the PR, meanwhile would you mind adding the following as well:
> 
> 1. Adagrad changed from 0.01 to 0.001
> 2. Adamax changed from 0.002 to 0.001
> 3. NAdam changed from 0.002 to 0.001
> 4. Adadelta changed from 1.000 to 0.001
> 
> Thanks!

Added - thanks!

**Comment by bananabowl** [comment]
> Why do we have the change against master?
> I thought these were for the patch release?

These were missed in the 1.14 relnotes, not changes in the patch release.

**Review by tanzhenyu — APPROVED** [review]

**Review by gunan — APPROVED** [review]
